src/tests/Microsoft.PowerFx.Core.Tests/LexerTests.cs

```diff
@@ -640,5 +641,16 @@ public void ITextFirstTokenTest()
             ITokenTextSpan tokenTextSpan = token;
             Assert.True(tokenTextSpan.CanBeHidden);
         }
+
+        [Fact]
+        public void TextFirstEmptyExpressionTest()
+        {
+            var engine = new Engine();
+            var parserOptions = new ParserOptions(CultureInfo.InvariantCulture) { TextFirst = true };
+            var result = engine.Check(string.Empty, parserOptions);
+
+            Assert.Equal(FormulaType.String, result.ReturnType);
+            Assert.True(result.IsSuccess);
+        }

```

Comment:
Can you also evaluate the expression? Would the result be blank, or an empty string? This test will be the _de facto_ specification for this scenario.